cmd/bosun/sched/sched.go

```diff
@@ -43,6 +43,7 @@ type Schedule struct {
 	LastCheck     time.Time
 	nc            chan interface{}
 	notifications map[*conf.Notification][]*State
+	lastLogTimes  map[expr.AlertKey]time.Time
```

Comment:
Thoughts about putting LastLogTime on the State type instead? Then it could survive restarts and we could avoid the potential need to lock the map.